docs-site/src/index.css

```
@@ -1084,6 +1086,228 @@ body {
+  font-size: 10px;
+  font-weight: 700;
+  letter-spacing: 0.08em;
+  color: var(--color-sage);
+  border: 1px solid currentColor;
+  border-radius: 999px;
+  padding: 2px 10px;
+  opacity: 0;
+  animation: outcomeIn 9s ease-in-out infinite;
+}
+
+@keyframes todoBoxFill {
```

Stylelint (17.14.0)

[error] Expected keyframe name "todoBoxFill" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)